geode-dunit/src/distributedTest/java/org/apache/geode/test/dunit/rules/tests/SharedCountersRuleDistributedTest.java

```diff
@@ -39,7 +43,7 @@
 @SuppressWarnings("serial")
 public class SharedCountersRuleDistributedTest implements Serializable {
 
-  private static final int TWO_MINUTES_MILLIS = 2 * 60 * 1000;
+  private static final long TWO_MINUTES_MILLIS = getTimeout().getValueInMS();
```

Comment:
Rename this to something like `DEFAULT_TIMEOUT` or `DEFAULT_AWAITILITY_TIMEOUT` or similar. Because 2 minutes is likely wrong.